src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Averin";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Averin";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.bigBar {
+  margin: 20px 0;
+}
+
+.input {
+  font-family: "Averin", sans-serif;
+  font-weight: 300;
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+}
+
+.input_big {
```

Comment:
move styles from this class to `.input`, leave only `input--small` to override values for small input (look into checklist p2)